package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "oppia",
+  "version": "2.8.1",
+  "description": "Oppia enables the creation of interactive online lessons.",
+  "main": "gulpfile.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/oppia/oppia.git"
+  },
+  "author": "",
+  "license": "Apache v2",
```

Comment:
Hi @seanlip ,

Can we change the license to `Apache-2.0` ? because this a valid SPDX License expr according to the list [here](https://spdx.org/licenses/). Changing the license will help us to remove such npm warnings mentioned below.

<img width="506" alt="Screenshot 2019-05-03 at 10 23 27 PM" src="https://user-images.githubusercontent.com/41017303/57152861-59f12580-6df2-11e9-98e5-b38f74dc96a8.png">

Reply:
Yes that's a good idea, thanks. Do you want to do it or shall I?

Reply:
Hi @seanlip , i'll take up this.